src/klicky.ts

```diff
@@ -1,3 +1,54 @@
 export const Klicky = () => {
   console.log("klickly ran!");
 };
+
+const outer = document.querySelector('.outer');
+
+const capture = {
+  capture : true
+};
+
+const noneCapture = {
+  capture : false 
+};
+
+const once = {
+  once : false 
+};
+
+outer?.addEventListener('click', onceHandler, once);
+
+function onceHandler(event)
```

Comment:
It looks like you're using older ES5 JS syntax here, this works fine, but there are some JS quirks that could be confusing down the line. Can you try re-writing this using an arrow function (we call it ES6 syntax)?

Reply:
Rewrote the function using an arrow function